example/lib/main.dart

```diff
@@ -233,3 +235,62 @@ class _MyAppState extends State<MyApp> {
     super.dispose();
   }
 }
+
+/*
+Test Id's from:
+https://developers.google.com/admob/ios/banner
+https://developers.google.com/admob/android/banner
+
+App Id
+Android: ca-app-pub-3940256099942544~3347511713
+iOS: ca-app-pub-3940256099942544~1458002511
+
+Banner
+Android: ca-app-pub-3940256099942544/6300978111
+iOS: ca-app-pub-3940256099942544/2934735716
+
+Interstitial
+Android: ca-app-pub-3940256099942544/1033173712
+iOS: ca-app-pub-3940256099942544/4411468910
+
+Reward Video
+Android: ca-app-pub-3940256099942544/5224354917
+iOS: ca-app-pub-3940256099942544/1712485313
+*/
+
+String getAppId() {
+  if (Platform.isIOS) {
+    return 'ca-app-pub-3940256099942544~1458002511';
+  } else if (Platform.isAndroid) {
+    return 'ca-app-pub-3940256099942544~3347511713';
+  }
+  return null;
+}
```

Comment:
All test `id`'s are different per platform. I was getting weird behavior's when I mixed and Android App Id with and iOS ad unit id. This was a simple fix. I'm open to cleaner suggestions.